src/main/java/org/generationcp/middleware/api/germplasmlist/data/GermplasmListDataServiceImpl.java

```diff
@@ -240,6 +240,21 @@ public void updateGermplasmListDataView(final Integer listId, final List<Germpla
 		this.daoFactory.getGermplasmListDAO().save(germplasmList);
 	}
 
+	@Override
+	public void reOrderEntries(final Integer listId, final List<Integer> selectedEntries, final Integer entryNumberPosition) {
+		this.daoFactory.getGermplasmListDataDAO().reOrderEntries(listId, selectedEntries, entryNumberPosition);
+	}
+
+	@Override
+	public long countByListId(final Integer listId) {
+		return this.daoFactory.getGermplasmListDataDAO().countByListId(listId);
+	}
+
+	@Override
+	public List<Integer> getLrecidsByListId(final Integer listId) {
```

Comment:
rename function to a more friendly name :)

Reply:
Done!